pyiceberg/catalog/__init__.py

```diff
@@ -631,9 +631,9 @@ def drop_view(self, identifier: Union[str, Identifier]) -> None:
         """
 
     @deprecated(
-        deprecated_in="0.8.0",
-        removed_in="0.9.0",
-        help_message="Support for parsing catalog level identifier in Catalog identifiers is deprecated. Please refer to the table using only its namespace and its table name.",
+        deprecate_in="0.8.0",
+        remove_in="0.9.0",
+        topic="Please refer to the table using only its namespace and its table name.",
```

Comment:
@ndrluis Sorry for not getting to this earlier. My main concern is that we might make things more complex than necessary. Why would we not use the `addendum` here? I find the `prefix` and `topic` not very intuitive, as in, it is not directly obvious to me how the deprecation warning will look like. That raises the question, should we reduce the number of options?

Reply:
I changed the topic attribute to 'addendum' and will improve the documentation to clarify the output message for each case.